io.sourceforge.pentobi.yml

```diff
@@ -1,6 +1,6 @@
 app-id: io.sourceforge.pentobi
 runtime: org.kde.Platform
-runtime-version: '5.15-21.08'
+runtime-version: 6.2
```

Comment:
6.4 is available.

Reply:
The game requested 6.2 so I pinned that.